smartseq2_single_sample/run_multi_samples/scripts/calculate_cost_workflow.py

```diff
@@ -0,0 +1,255 @@
+import argparse
+import json
+import math
+import urllib.request, urllib.error, urllib.parse
+import base64
```

Comment:
This import isn't being used so it can be removed

Reply:
yes, remove base64, sys, re